lexers/u/usd.go

```diff
@@ -0,0 +1,18 @@
+package u
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// USD lexer.
+var USD = internal.Register(MustNewLexer(
```

Comment:
May we follow the same variable name as pygments? In this case would be `Usd`?
https://github.com/pygments/pygments/blob/a590ac5ea7c00a41e253834306bfa19e38349c0b/pygments%2Flexers%2Fusd.py#L31

Reply:
What is your intention here? To ensure we always name it identical to pygments?

For me `USD` feels more natural in go. Also it matches the lexer name and with other lexers variable names also differ (https://github.com/wakatime/chroma/blob/j-lexers/lexers/j/jsonld.go#L9, https://github.com/wakatime/chroma/blob/j-lexers/lexers/j/json.go#L9)

Reply:
JSON must be all upper case because of linting in Go. For others I think we could port the same as it is in Pygments. For example, `Java Server Page` is named `Jsp` in Pygments and in Chroma I ported as `Jsp` and not `JavaServerPage`. So it's a tricky decision here. To be honest I couldn't find any guideline for this migration. Let's don't take care of character cases in our PRs.

Reply:
Sounds good. In the end, these are variable names, which do not make a functional difference.